tests/virt/cluster/common_templates/general/test_base_template.py

```
@@ -131,10 +142,11 @@ def get_centos_templates_list():
 
 @pytest.fixture()
 def common_templates_expected_list():
-    common_templates_list = get_rhel_templates_list()
-    common_templates_list += get_fedora_templates_list()
-    common_templates_list += get_windows_templates_list()
-    common_templates_list += get_centos_templates_list()
+    node_cpu_arch = infra.get_nodes_cpu_architecture()
```

Pylint (3.3.7)

[error] No value for argument 'nodes' in function call

(E1120)